extractor/src/main/scala-sbt-1.3/sbt/jetbrains/keysAdapterEx.scala

```diff
@@ -1,10 +1,33 @@
 package sbt.jetbrains
 
-import sbt.{Def, File, Task, TaskKey}
+import lmcoursier.definitions.CacheLogger
+import sbt.KeyRanks.{DTask, Invisible}
+import sbt.{Def, File, Global, GlobalScope, Keys, Logger, Setting, Task, TaskKey}
 
 object keysAdapterEx {
   //NOTE: sbt.Keys.scalaCompilerBridgeBinaryJar exists since SBT 1.2.3, so we detect it only since 1.3.0
   val myScalaCompilerBridgeBinaryJar: Def.Initialize[Task[Option[File]]] = Def.taskDyn {
     sbt.Keys.scalaCompilerBridgeBinaryJar
   }
+  val artifactDownloadTask: TaskKey[Option[CacheLogger]] = TaskKey("artifactDownloadTask", rank = DTask)
+  lazy val artifactDownloadCsrLogger: Def.Initialize[Task[Option[CacheLogger]]] = Def.task {
+    val st = Keys.streams.value
+    Some(new CoursierLogger(st.log))
```

Comment:
in the `csrLogger` task implementation in the sbt, the if condition is performed whether super shell is enabled. Why you removed it here?